department-of-veterans-affairs/caseflow
# pbradin/CASEFLOW-214 Permissions for Edit Nod Date link

### Description
The ability to edit NOD dates needs to be restricted to BvaIntake.singleton.users and ClerkOfTheBoard.singletone.users.  Displaying the "Edit NOD Date" link in the Queue Timeline section needs to be restricted to those two user types.

### Acceptance Criteria
- [ ] NOD exists and the "Edit NOD Date" link displays for ClerkOfTheBoard and BvaIntake users.
- [ ] NOD exists and the "Edit NOD Date" link does not display for other user types.

### Testing Plan
1. Login to Queue as "COB_User" and/or "BVADWISE"
2. Make sure "edit_nod_date" featuretoggle is enabled.
3. Choose an appeal that has an existing NOD and confirm that the link displays.
4. Switch user to someone without permission (like BVAAABSHIRE)
5. Choose an appeal that has an existing NOD and confirm that the link does not display.

### User Facing Changes
 - [ ] Screenshots of link displaying and not displaying depending on user

 WITH PERMISSIONS
![cob_user_with_edit_link](https://user-images.githubusercontent.com/23639439/101190986-1f331d00-3627-11eb-87d3-8d3896a7b435.jpg)

WITHOUT PERMISSIONS
![abshire_no_edit_link](https://user-images.githubusercontent.com/23639439/101191034-2b1edf00-3627-11eb-9fbd-0a1d40b5412b.jpg)
